applications/nrf5340_audio/src/audio/audio_sync_timer.c

```diff
@@ -64,10 +65,16 @@ LOG_MODULE_REGISTER(audio_sync_timer, CONFIG_LOG_AUDIO_SYNC_TIMER_LEVEL);
 
 #define AUDIO_SYNC_TIMER_NET_APP_IPC_EVT NRF_IPC_EVENT_RECEIVE_4
 
+#define AUDIO_SYNC_TIMER_IRQ_PRIORITY 6
+
 static const nrfx_timer_t timer_instance = NRFX_TIMER_INSTANCE(AUDIO_SYNC_TIMER_INSTANCE);
 
+#define SYNC_LED DT_GPIO_PIN(DT_NODELABEL(led1_blue), gpios)
```

Comment:
One thing I missed:
We like to do all #define before declaring variables. So move this to over static const nrfx_timer_t